data/migrations/V0091__drop_ofec_sched_b_objects.sql

```diff
@@ -0,0 +1,196 @@
+
+-- ------------------
+-- 	functions and triggers
+-- ------------------
+DROP TRIGGER IF EXISTS nml_sched_b_after_trigger ON disclosure.nml_sched_b;
+DROP TRIGGER IF EXISTS nml_sched_b_before_trigger ON disclosure.nml_sched_b;
+
+DROP TRIGGER IF EXISTS f_item_sched_b_before_trigger ON disclosure.f_item_receipt_or_exp;
+DROP TRIGGER IF EXISTS f_item_sched_b_after_trigger ON disclosure.f_item_receipt_or_exp;
+
+-- trigger functions
+drop function IF EXISTS public.ofec_sched_b_delete_update_queues();
+drop function IF EXISTS public.ofec_sched_b_insert_update_queues();
+drop function IF EXISTS public.ofec_sched_b_update_queues();
+
+-- functions
+drop function IF EXISTS public.ofec_sched_b_update_fulltext();
+drop function IF EXISTS public.ofec_sched_b_update();
+
+drop function IF EXISTS public.drop_old_itemized_schedule_b_indexes(numeric, numeric); 
+
+-- ------------------
+-- Function: public.add_partition_cycles(numeric, numeric)
+-- ------------------
+CREATE OR REPLACE FUNCTION public.add_partition_cycles(
+    start_year numeric,
+    amount numeric)
+  RETURNS void AS
+$BODY$
+
+DECLARE
+
+    first_cycle_end_year NUMERIC = get_cycle(start_year);
+
+    last_cycle_end_year NUMERIC = first_cycle_end_year + (amount - 1) * 2;
+
+    master_table_name TEXT;
+
+    child_table_name TEXT;
+
+    schedule TEXT;
+
+BEGIN
+
+    FOR cycle IN first_cycle_end_year..last_cycle_end_year BY 2 LOOP
+
+        FOREACH schedule IN ARRAY ARRAY['a'] LOOP
+
+            master_table_name = format('ofec_sched_%s_master', schedule);
+
+            child_table_name = format('ofec_sched_%s_%s_%s', schedule, cycle - 1, cycle);
+
+            EXECUTE format('CREATE TABLE %I (
+
+                CHECK ( two_year_transaction_period in (%s, %s) )
+
+            ) INHERITS (%I)', child_table_name, cycle - 1, cycle, master_table_name);
+
+
+
+        END LOOP;
+
+    END LOOP;
+
+    PERFORM finalize_itemized_schedule_a_tables(first_cycle_end_year, last_cycle_end_year, FALSE, TRUE);
+ 
+END
+
+$BODY$
+  LANGUAGE plpgsql VOLATILE
+  COST 100;
+ALTER FUNCTION public.add_partition_cycles(numeric, numeric)
+  OWNER TO fec;
+
+
+DROP FUNCTION IF EXISTS public.finalize_itemized_schedule_b_tables(numeric, numeric, boolean, boolean);
+
+-- ------------------
+-- FUNCTION disclosure.finalize_itemized_schedule_b_tables
+-- ------------------
+CREATE OR REPLACE FUNCTION disclosure.finalize_itemized_schedule_b_tables(

```

Comment:
Should this function be dropped?

Reply:
This function had been rewritten to work on the new set of fec_fitem_sched_b tables.